.github/workflows/builds.yml

```diff
@@ -84,6 +89,7 @@ jobs:
         working-directory: suricata
         run: python3 ../run.py --quiet
       - name: Running check-eve
+        if: github.ref == 'refs/heads/master'
```

Comment:
Is this needed?

Reply:
Yes because master-6.0.x has some additional fields (for dns if I remember correctly).
You can check the previous CI runs...
Or run it after running S-V tests with suricata 6.0.4

Reply:
But isn't this checking the ref of suricata-verify itself, not the ref of Suricata.

Reply:
Indeed...
Force-pushing another version to test CI